test/0.8.25/vaults/delegation/delegation.test.ts

```diff
@@ -56,8 +58,9 @@ describe("Delegation.sol", () => {
     weth = await ethers.deployContract("WETH9__MockForVault");
     wsteth = await ethers.deployContract("WstETH__HarnessForVault", [steth]);
     hub = await ethers.deployContract("VaultHub__MockForDelegation", [steth]);
+    lidoLocator = await ethers.deployContract("LidoLocator__HarnessForDashboard", [steth, wsteth]);
```

Comment:
Please provide the same feedback regarding the locator as in the dashboard tests.